docs/best-practices/continuous-integration/circle-ci.md

```diff
@@ -55,7 +55,7 @@ orbs:
 jobs:
   build:
     macos:
-      xcode: "13.1.0"
+      xcode: "15.4.0"
```

Comment:
Currently [v2.222 tests](https://github.com/fastlane/fastlane/blob/e1ab951b63aeb7a669b415708ffbe0e1346c0f59/.circleci/config.yml#L282-L285) agiainst 15.3.0 so these should probably be kept in sync.

Reply:
(i.e. once 15.4 is also added there I'm all for having the most uptodate here, given the examples have no ruby version constraints too that would make the CircleCI env angry…)